packages/fxa-auth-server/lib/payments/stripe.ts

```diff
@@ -558,37 +535,20 @@ class StripeHelper {
     email: string
   ): Promise<Stripe.Customer | undefined> {
     try {
-      return await this.customer(uid, email, true);
+      return await this.customer({ uid, email, forceRefresh: true });
     } catch (err) {
       this.log.error(`subhub.refreshCachedCustomer.failed`, { err });
       return;
     }
   }
 
-  /**
-   * Build a key used for Redis cache based on user ID & email.
-   */
-  customerCacheKey(uid: string, email: string): string {
-    return `customer-${uid}|${email}`;
-  }
-
   /**
    * Remove the cache entry for a customer account
    * This is to be used on account deletion
    */
+  @CacheClear({ cacheKey: StripeHelper.customerCacheKey })
```

Comment:
Runs the function inside, and if it doesn't throw, will clear the cache key here per https://github.com/joshuaslate/type-cacheable#cacheclear